backend/typescript/prisma/schema.prisma

```diff
@@ -12,11 +12,21 @@ model test {
 }
 
 model users {
-   id        String @id @default(auto()) @map("_id") @db.ObjectId
-   v         Int    @map("__v")
-   authId    String
-   email     String
-   firstName String
-   lastName  String
-   role      String
- }
\ No newline at end of file
+  id        String @id @default(auto()) @map("_id") @db.ObjectId
+  v         Int    @map("__v")
+  authId    String
+  email     String
+  firstName String
+  lastName  String
+  role      String
+}
+
+model volunteerSignUp {
+  id        String @id @default(auto()) @map("_id") @db.ObjectId
+  v         Int    @map("__v")
+  email     String
+  firstName String
+  lastName  String
+  admin_id  String
+  status    String
```

Comment:
make this Status an ENUM type. https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#enum